### ISSUE TITLE
TryFrom/TryInto are not resolved from the prelude in 2021 edition

### ISSUE THREAD

**Issue by Veetaha** [rust-lang/rust-analyzer#13250]
# Reproduction

1. Run `cargo new repro`, assuming cargo sets `edition = "2021"` in `Cargo.toml`.
2. Put this code into `main.rs`:
   ```rust
   fn main() {
       let _: bool = TryFrom::try_from(true).unwrap();
       let _: bool = true.try_into().unwrap();
   }
   ```
3. Hover over `TryFrom::try_from` and `try_into` symbols and observe them as unresolved according to rust-analyzer: <img width="406" alt="image" src="https://user-images.githubusercontent.com/36276403/190881593-f853291b-3021-45c9-ac21-dcf28c6423a3.png">

**rust-analyzer version**: `rust-analyzer version: 0.3.1203-standalone`
**rustc version**: `rustc 1.63.0 (4b91a6ea7 2022-08-08)`

**Comment by lnicola** [comment]
Fixed in #13235 or another recent PR:

![image](https://user-images.githubusercontent.com/308347/190887149-b3a1e671-81d6-4171-a98c-43b29232cae9.png)

**Comment by lnicola** [comment]
Sorry, that was probably #13147.
